chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -820,12 +820,14 @@ sync.error.sslConnectionError 		= SSL connection error
 sync.error.checkConnection 			= Error connecting to server. Check your Internet connection.
 sync.error.emptyResponseServer 		= Empty response from server.
 sync.error.invalidCharsFilename		= The filename '%S' contains invalid characters.\n\nRename the file and try again. If you rename the file via the OS, you will need to relink it in Zotero.
+sync.error.apiKeyInvalid			= We could not authenticate your account with Zotero servers. Please re-enter your account details.
 
 sync.lastSyncWithDifferentAccount	= This Zotero database was last synced with a different zotero.org account ('%1$S') from the current one ('%2$S').
 sync.localDataWillBeCombined		= If you continue, local Zotero data will be combined with data from the '%S' account stored on the server.
 sync.localGroupsWillBeRemoved1		= Local groups, including any with changed items, will also be removed from this computer.
 sync.avoidCombiningData				= To avoid combining or losing data, revert to the '%S' account or use the Reset options in the Sync pane of the Zotero preferences.
 sync.localGroupsWillBeRemoved2		= If you continue, local groups, including any with changed items, will be removed and replaced with groups linked to the '%1$S' account.\n\nTo avoid losing local changes to groups, be sure you have synced with the '%2$S' account before syncing with the '%1$S' account.
+sync.unlinkWarning					= Are you sure you want to unlink this account from Zotero?\n\nThis will stop syncing your database but no data will be lost.
```

Comment:
"Are you sure you want to unlink this account?\n\n%S will no longer sync your data, but your data will remain locally."

And pass in `Zotero.appName`. (We haven't generally done this historically, but we want to start not hard-coding "Zotero" anywhere, for the sake of third-party builds.)

(When we make this fancier and give options, let's remember to make groups read-only after unlinking if people choose to keep them.)